GETTING_STARTED.md

```diff
@@ -0,0 +1,416 @@
+# Getting started
+
+In this document you will find a step-by-step guide on how to get Eunomia running in a local Minikube cluster.
+
+The resources Eunomia, the GitOps controller, is watching and how it is done will be discussed in general.
```

Comment:
Rephrase. Not sure what you're trying to say here.

Reply:
+1